onnx/defs/tensor/defs.cc

```diff
@@ -1301,11 +1301,17 @@ ONNX_OPERATOR_SET_SCHEMA(
         }));
 
 static const char* Unsqueeze_ver11_doc = R"DOC(
-Insert single-dimensional entries to the shape of a tensor.
-Takes one required argument `axes`, a list of dimensions that will be inserted.
-Dimension indices in `axes` are as seen in the output tensor. For example:
-  Given a tensor such that tensor with shape [3, 4, 5], then
-  Unsqueeze(tensor, axes=[0, 4]) has shape [1, 3, 4, 5, 1]
+Insert single-dimensional entries to the shape of an input tensor (`data`).
+Takes one required argument `axes` - which contains a list of dimension indices and this operator will insert a dimension of value `1` into the corresponding index of the output tensor (`expanded`).
+
+For example:
+  Given an input tensor (`data`) of shape [3, 4, 5], then
+  Unsqueeze(data, axes=[0, 4]) outputs a tensor (`expanded`) containing same data as `data` but with shape [1, 3, 4, 5, 1].
```

Comment:
As discussed, we should consider adding a note clarifying that the order of axes specified in the `axes` attribute does not matter, and that they should be sorted before being applied.

Reply:
I added this line - 

The order of values in `axes` does not matter and can come in any order.

I think this is good for a high level spec. Details like the operator will sort them after correcting negative values are superfluous, I feel. What do you think ?